pkg/dashboard/resource/project.go

```diff
@@ -39,6 +43,30 @@ type projectInfo struct {
 	Spec *platform.ProjectSpec `json:"spec,omitempty"`
 }
 
+type functionImportMap map[string]*functionInfo
+
+func (fim *functionImportMap) UnmarshalJSON(data []byte) error {
+	var functionImports map[string]*struct {
+		Function *functionInfo
+	}
```

Comment:
Nope - because the export exports for each function:
```
"<functionName>": {
  "function": ...
  "functionEvents": ...
}
```
So I am explicitly unmarshalling the `function` part